1_fetch/src/get_data.R

```diff
@@ -0,0 +1,7 @@
+# Get the data from ScienceBase
+get_data_from_sb <- function(sbid, outer_dir){
+  data_filepath <- file.path(outer_dir, '1_fetch', 'out', 'model_RMSEs.csv')
+  item_file_download(sbid, names = 'me_RMSE.csv', destinations = data_filepath, overwrite_file = TRUE)
+  
+  return(data_filepath)
```

Comment:
Also nice use of a return value. This pattern of returning the filename for a function which creates a file is one which is used a bunch by `targets`, so you'll be right at home in `targets`.